Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gemspec
```

Comment:
Can you add this snippet below this: https://github.com/logstash-plugins/logstash-input-beats/blob/master/Gemfile#L5-L11

This allows for testing against different branch versions.

Reply:
Also, can you add a .travis file like this: https://github.com/logstash-plugins/logstash-input-beats/blob/master/.travis.yml

Reply:
@jakelandis Should both these changes be made under https://github.com/elastic/logstash/tree/master/lib/pluginmanager/templates as well for future runs of the plugin generator (`bin/logstash-plugin generate ...`)?

Reply:
Yes, and thank you for https://github.com/elastic/logstash/pull/9193